### PR TITLE
[SCB-897] Support max initial line length config

### PR DESCRIPTION
**Pull request by yhs0092**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---
See details in [SCB-897](https://issues.apache.org/jira/browse/SCB-897)

### CODE REVIEW

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/ConsumerMain.java [hunk 1]
```diff
@@ -110,6 +113,9 @@ private static void testStandalone() throws Throwable {
     ITJUnitUtils.runWithHighwayAndRest(TestTrace.class);
     ITJUnitUtils.run(TestTraceEdge.class);
     ITJUnitUtils.runWithHighwayAndRest(TestParamCodec.class);
+    ITJUnitUtils.run(TestParamCodecEdge.class);
+
+    ITJUnitUtils.runWithTransports(Collections.singletonList(Const.RESTFUL), TestRestServerConfig.class);
```
Comment (wujimin) [inline integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/ConsumerMain.java]: create a new method: runWithRest

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/base/TestRestServerConfig.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      RestServerConfigSchemaIntf.class);
+
+  @BeforeClass
+  public static void classSetup() {
+    consumers.init(ITJUnitUtils.getTransport());
+  }
+
+  @Test
+  public void testMaxInitialLineLength_underLimit() {
+    String querySegment = "1234567890";
```
Comment (wujimin) [inline integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/base/TestRestServerConfig.java]: try Strings.repeat

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/base/TestRestServerConfig.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    StringBuilder q = new StringBuilder();
+    while (q.length() < 4900) {
+      q.append(querySegment);
+    }
+    String result = consumers.getIntf().testMaxInitialLineLength(q.toString());
+    Assert.assertEquals("OK", result);
+  }
+
+  @Test
+  public void testMaxInitialLineLength_exceedLimit() {
```
Comment (wujimin) [inline integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/base/TestRestServerConfig.java]: test 4999 and 5000?

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/ConsumerMain.java (hunk 1 above)
Comment (yhs0092) [reply integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/ConsumerMain.java]: Fixed. I find there is already a runWithRest method in ITJunitUtils.

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/base/TestRestServerConfig.java (hunk 2 above)
Comment (yhs0092) [reply integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/base/TestRestServerConfig.java]: Fixed

File: integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/base/TestRestServerConfig.java (hunk 3 above)
Comment (yhs0092) [reply integration-tests/it-consumer/src/main/java/org/apache/servicecomb/it/testcase/base/TestRestServerConfig.java]: Fixed, and there is conflict between this pr and master, so I rebase this branch.

**Pull request merged**